deploy/eck-operator/templates/statefulset.yaml

```diff
@@ -117,6 +117,10 @@ spec:
         {{- with .Values.volumes }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
+      {{- if .Values.webhook.hostNetwork }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      {{- end }}
```

Comment:
I initially read `hostNetwork: true` without the preceding `{{- if .Values.webhook.hostNetwork }}`, but I see now what I first was mis-reading. 👍